include/rosdiscover-clang/Ast/Stmt/SymbolicExpr.h

```diff
@@ -152,6 +169,11 @@ class BinaryExpr : public SymbolicExpr {
     };
   }
 
+  std::vector<const SymbolicExpr*> getChildren() const override {
+    return {expr1.get(), expr2.get()};
```

Comment:
rename `expr1` and `expr2` to lhs and rhs

Reply:
Fixed